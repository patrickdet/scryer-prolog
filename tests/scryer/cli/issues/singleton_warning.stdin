['singleton_example.pl'].
halt.
