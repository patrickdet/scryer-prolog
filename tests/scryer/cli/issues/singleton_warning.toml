# issue 812
args = ["-f", "--no-add-history"]
