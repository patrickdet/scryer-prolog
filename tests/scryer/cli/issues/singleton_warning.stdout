% Warning: singleton variables X at line 4 of singleton_example.pl
   true.
